data/human/news.txt

```diff
@@ -1508,13 +1605,29 @@ news "tourist pa system"
 			"Dear travelers: the spaceport is a strictly no-smoking zone. Thank you for your consideration."
 			"Dear travelers: welcome to our planet. Please remember to synchronize your personal clocks to the local time and night/day cycle duration."
 			"Please stand clear of the doors when the shuttle train is departing."
-			"This is the final boarding call for passengers Eline, John, and Robert Phillips. Please proceed to platform 3 for immediate boarding."
+			"This is the final boarding call for passengers ${tourist names}, ${tourist names}, and ${tourist names} ${tourist names}. Please proceed to platform ${platform number} for immediate boarding."
```

Comment:
The replacement for `Robert Phillips` seems a little off? The phrases `male names` and `female names` are all first names so perhaps there is something else you could use for the last name?

Reply:
Maybe start with a `"last names"` section in names.txt? (And just start by including Philips as last name.)

Reply:
I've added the last names from https://github.com/endless-sky/endless-sky/issues/6968 to this PR.